interfaces/IBF-dashboard/src/app/services/api.service.ts

```diff
@@ -198,7 +197,16 @@ export class ApiService {
         countryCode: countryCodeISO3,
         status,
         pcode,
-        event,
+      },
+      false,
+    ).toPromise();
+  }
+
+  closeEventPlaceCode(eventPlacecodeId: number) {
```

Comment:
`eventPlacecodeId` to `eventPlaceCode`

unless there is a difference between the `Place Code Id` and `Place Code` in which case use `eventPlaceCodeId` instead of `eventPlacecodeId`